csharp/test/IceSSL/configuration/PlatformTests.cs

```diff
@@ -100,7 +100,9 @@ private static void clientRejectServerUsingValidationCallback(TestHelper helper,
         };
         using var clientCommunicator = createClient(clientOptions);
 
-        var obj = ServerPrxHelper.createProxy(clientCommunicator, "server:" + helper.getTestEndpoint(10, "ssl"));
+        var obj = ServerPrxHelper.createProxy(
+            clientCommunicator,
+            "server:" + helper.getTestEndpoint(10, "ssl") + " -h 127.0.0.1");
```

Comment:
It's not clear to me how the original code - with no host - works.

All the communicators created by this test don't get any of the properties specified on the command line, in particular no Ice.Default.Host.

I would also prefer using localhost (in particular, to test ipv6), but localhost doesn't work in our devcontainer.

Reply:
> It's not clear to me how the original code - with no host - works.

For an outgoing connection the default is to use the loopback addresses

> If host is not specified in a proxy, Ice uses the value of the Ice.Default.Host property. If that property is not defined, outgoing connections are only attempted over loopback interfaces.

https://github.com/zeroc-ice/ice/blob/2846e40a8ea07d870d762d89b2075071ab04aba2/csharp/src/Ice/Internal/Network.cs#L706-L725

> All the communicators created by this test don't get any of the properties specified on the command line, in particular no Ice.Default.Host.

I guess this is because the creation of the first communicator consumes the command line args

https://github.com/zeroc-ice/ice/blob/2846e40a8ea07d870d762d89b2075071ab04aba2/csharp/test/IceSSL/configuration/Client.cs#L15-L22